doc/development.rst

```diff
@@ -27,32 +27,46 @@ your system with application only dependencies while you code.
 Vagrant
 -------
 
-.. todo::
+A ``Vagrantfile`` is provided which will give you a full local development setup.
 
-    Currently not ported to new architecture.
+To run the vagrant development environment make sure you have a supported
+virtualization system, vagrant installed, and have all commissaire project checked
```

Comment:
"Project" should be plural here, right?  And not sure if "commissaire-related projects" would be clearer?